documents/src/pages/elements/overlay-menu.md

```diff
@@ -31,46 +31,47 @@ section {
 <section>
   <ef-button cta id="button" aria-haspopup="true">Choose Item</ef-button>
 </section>
-<ef-overlay-menu id="menu" opened>
-  <ef-item type="header">Regions</ef-item>
-  <ef-item for="emea">EMEA</ef-item>
-  <ef-item for="n-america">N. America</ef-item>
-  <ef-item for="apac">APAC</ef-item>
-  <ef-item for="latin-america">Latin America</ef-item>
-  <ef-item type="header">Favorites</ef-item>
-  <ef-item icon="flame">Thailand</ef-item>
-  <ef-item type="divider"></ef-item>
-  <ef-item icon="dislike-empty" disabled>Unspecified</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="emea">
-  <ef-item>Spain</ef-item>
-  <ef-item>France</ef-item>
-  <ef-item for="united-kingdom">United Kingdom</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="united-kingdom">
-  <ef-item>Cardiff</ef-item>
-  <ef-item>Edinburgh</ef-item>
-  <ef-item>London</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="n-america">
-  <ef-item>Canada</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="apac">
-  <ef-item>China</ef-item>
-  <ef-item>Australia</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="latin-america">
-  <ef-item>Mexico</ef-item>
-  <ef-item>Brazil</ef-item>
-</ef-overlay-menu>
-
+<div class="container">
+  <ef-overlay-menu id="menu" opened>
+    <ef-item type="header">Regions</ef-item>
+    <ef-item for="emea">EMEA</ef-item>
+    <ef-item for="n-america">N. America</ef-item>
+    <ef-item for="apac">APAC</ef-item>
+    <ef-item for="latin-america">Latin America</ef-item>
+    <ef-item type="header">Favorites</ef-item>
+    <ef-item icon="flame">Thailand</ef-item>
+    <ef-item type="divider"></ef-item>
+    <ef-item icon="dislike-empty" disabled>Unspecified</ef-item>
+  </ef-overlay-menu>
+  <ef-overlay-menu id="emea">
+    <ef-item>Spain</ef-item>
+    <ef-item>France</ef-item>
+    <ef-item for="united-kingdom">United Kingdom</ef-item>
+  </ef-overlay-menu>
+  <ef-overlay-menu id="united-kingdom">
+    <ef-item>Cardiff</ef-item>
+    <ef-item>Edinburgh</ef-item>
+    <ef-item>London</ef-item>
+  </ef-overlay-menu>
+  <ef-overlay-menu id="n-america">
+    <ef-item>Canada</ef-item>
+  </ef-overlay-menu>
+  <ef-overlay-menu id="apac">
+    <ef-item>China</ef-item>
+    <ef-item>Australia</ef-item>
+  </ef-overlay-menu>
+  <ef-overlay-menu id="latin-america">
+    <ef-item>Mexico</ef-item>
+    <ef-item>Brazil</ef-item>
+  </ef-overlay-menu>
+</div>
 ```
 ::
 
 `ef-overlay-menu` is an overlay window that supports single-level and multi-level menus. It can be positioned by attaching to other elements, or its vertical and horizontal offset can be adjusted, if needed.
 
 ## Usage
-Create `ef-overlay-menu` with `ef-item` elements as menu items. Listen for the `item-trigger` event to identify a clicked item.
+Create `ef-overlay-menu` with `ef-item` elements as menu items. Listen for the `item-trigger` event to identify a clicked item. Furthermore, `item-trigger` event should listen at parent element of `ef-overlay-menu`.
```

Comment:
Let's add a new paragraph describing how to add event listener to a wrapper element when overlay-menus are nested.

Reply:
Also, this is one of the 2 live examples that a wrapper element is needed.

Reply:
Updated!

Reply:
I think we should 
* Describe about wrapper element in "Nested menu" section instead. Plus, add an inline link to "Managing selection" section for a live example.
* As the live example in "Usage" contains only one Overlay Menu, there should be no wrapper there.

Reply:
Thank you for your feedback. I have already refactored this section regarding your advice.